sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_generated/aio/_client.py

```diff
@@ -23,11 +24,11 @@ class KeyVaultClient(KeyVaultClientOperationsMixin):  # pylint: disable=client-a
     Vault service.
 
     :ivar role_definitions: RoleDefinitionsOperations operations
-    :vartype role_definitions: key_vault_client.aio.operations.RoleDefinitionsOperations
+    :vartype role_definitions: azure.keyvault.v7_5.aio.operations.RoleDefinitionsOperations
```

Comment:
It seems like v7_5 should be administration? Is this what got generated?

Reply:
This is what got generated, yeah... that's really odd. It does reflect the structure of the generated output though (sorry about the tiny screenshot):
<img width="311" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/39780829/83231e2f-1dc4-446d-b39c-81a926ce1b4d">

Reply:
It looks like maybe it's coming from the README: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/data-plane/readme.python.md?plain=1#L41

If this isn't publicly exposed anywhere it's probably fine, just was kind of strange to me when I saw it.